.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,54 @@
+name: Bug Report
+description: Create a report to help us reproduce and fix the bug
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      #### Thank you for contributing! Before reporting a bug, please make sure the issue has no duplicate and hasn't been already addressed by searching through [the existing and past issues](https://github.com/opencv/opencv/issues?page=1&q=is%3Aissue+sort%3Acreated-desc). To test with the latest OpenCV, you could either build from source or try with opencv-python via `pip install opencv-python-rolling`.
```

Comment:
The line is too long.

Reply:
I propose to add link to FAQ at least.

Reply:
just updated, please have a look: https://github.com/fengyuentau/opencv/issues/new?assignees=&labels=&template=bug_report.yml

Reply:
just updated, please have a look: https://github.com/fengyuentau/opencv/issues/new?assignees=&labels=&template=bug_report.yml